aexpect/remote_door.py

```diff
@@ -838,7 +896,7 @@ def import_remote_exceptions(exceptions=None, modules=None):
     :type exceptions: [str] or None
     :param modules: full module paths whose custom exceptions will first be
                     detected and then automatically imported (optional)
-    :type exceptions: [str] or None
+    :type modules: [str] or None
```

Comment:
good catch, ideally should be separate commit (but unless there are more we can close one eye and let it here ;-) )

Reply:
I don't think it relates to a separate commit since the changes of exposing extra classes are very similar in nature to the changes of serializing extra exceptions as can be seen from the function signatures as well as the corresponding tests.

Reply:
What I meant is that this is a typo-fix and not actual change related to this commit.

Reply:
Ok, done and separated so I will resolve this.